ui/src/app/routing/app-routing.tsx

```diff
@@ -137,59 +123,17 @@ const loadErrorReporter = () => {
 };
 
 export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = () => {
-  const {authLoaded = false} = useStore(authStore);
+  const {authLoaded} = useAuthentication();
   const isUserDisabled = useIsUserDisabled();
   const [pollAborter, setPollAborter] = useState(new AbortController());
-  const [isSignedIn, setIsSignedIn] = useState(false);
   const [, setTestAccessTokenOverride] = useState(undefined);
-  const [signInMounted, setSignInMounted] = useState(false);
-  const [doSignOut, setDoSignOut] = useState(false);
   const [isCookiesEnabled, setIsCookiesEnabled] = useState(false);
   const [overriddenUrl, setOverriddenUrl] = useState('');
 
-  // TODO angular2react - does this work?
-  const onSignIn = (): void => {
-    setSignInMounted(true);
-  };
-
-  const signIn = (): void => {
-    AnalyticsTracker.Registration.SignIn();
-
-    gapi.auth2.getAuthInstance().signIn({
-      'prompt': 'select_account',
-      'ux_mode': 'redirect',
-      'redirect_uri': `${window.location.protocol}//${window.location.host}`
-    });
-  };
-
-  const signOut = (): void => {
-    // If we're in puppeteer, we never call gapi.auth2.init, so we can't sign out normally.
-    // Instead, we revoke all the access tokens and reset all the state.
-    if (environment.allowTestAccessTokenOverride && window.localStorage.getItem(LOCAL_STORAGE_KEY_TEST_ACCESS_TOKEN)) {
-      authStore.set({...authStore.get(), authLoaded: true, isSignedIn: false});
-      setIsSignedIn(false);
-      window.setTestAccessTokenOverride('');
-    } else {
-      gapi.auth2.getAuthInstance().signOut();
-    }
-  };
-
-  const subscribeToInactivitySignOut = () => {
```

Comment:
except for this one, which I renamed to `enableInactivityTimeout` and stuck in `authStore`